cmd/schemagen/schemagen.go

```diff
@@ -120,8 +124,8 @@ func renderTemplate(md *gocql.KeyspaceMetadata) ([]byte, error) {
 			if c.Type == "decimal" && !existsInSlice(imports, "gopkg.in/inf.v0") {
 				imports = append(imports, "gopkg.in/inf.v0")
 			}
-			if c.Type == "duration" && !existsInSlice(imports, "github.com/gocql/gocql") {
-				imports = append(imports, "github.com/gocql/gocql")
```

Comment:
Why adding `v2` to gocql? I thought we add `v2` to `gocqlx` not `gocql`.

Reply:
Fixed, thanks.